packages/near-fast-auth-signer-e2e-tests/tests/signMultiChain/sign-multi-chain.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { KeyPair } from '@near-js/crypto';
+import { expect, Page, test } from '@playwright/test';
+
+import {
+  receivingAddresses, getFastAuthIframe
+} from '../../utils/constants';
+import { overridePasskeyFunctions } from '../../utils/passkeys';
+import SignMultiChain from '../models/SignMultiChain';
+
+let page: Page;
+let signMultiChain: SignMultiChain;
+
+const userFAK = process.env.MULTICHAIN_TEST_ACCOUNT_FAK;
+const accountId = process.env.MULTICHAIN_TEST_ACCOUNT_ID;
+
+const fakKeyPair = KeyPair.fromString(userFAK);
+
+const isAuthenticated = async (loggedIn: boolean) => {
+  if (!page) return;
+  if (loggedIn) {
+    await overridePasskeyFunctions(page, {
+      creationKeypair:  fakKeyPair,
+      retrievalKeypair: fakKeyPair
+    });
+    return;
+  }
+  await overridePasskeyFunctions(page, {
+    creationKeypair:  KeyPair.fromRandom('ed25519'),
+    retrievalKeypair: KeyPair.fromRandom('ed25519')
+  });
+};
+
+test.describe('Sign MultiChain', () => {
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    signMultiChain = new SignMultiChain(page);
+    await page.goto('/');
+    await page.waitForLoadState('domcontentloaded');
+    // Disable pointer events that will interrupt click event on "Approve button"
+    await signMultiChain.disablePointerEventsInterruption();
+    await page.evaluate(
+    // eslint-disable-next-line no-shadow
+      async ([accountId]) => {
+        window.localStorage.setItem('accountId', JSON.stringify(accountId));
+      },
+      [accountId]
+    );
+  });
+
+  test.beforeEach(() => {
+    test.slow();
+  });
+
+  const ensureWalletSelectorIsLoaded = async () => {
+    const walletSelector = page.locator('#ws-loaded');
+    await expect(walletSelector).toBeVisible();
+  };
```

Comment:
there is a util function called `isWalletSelectorLoaded` inside `walletSelector.ts` that can replace this